tests/test_cli.py

```diff
@@ -71,6 +71,22 @@ def test_cli_path_mapping(
         builder.cursor.execute.assert_called()
 
 
+@mock.patch("pyathena.connect")
+def test_count_builder_mapping(mock_connect):  # pylint: disable=unused-argument
+    with does_not_raise():
+        builder = cli.main(
+            cli_args=[
+                "build",
+                "-t",
+                "study_python_counts_valid",
+                "-s" "./tests/test_data",
+                "--database",
+                "test",
+            ]
+        )
+        builder.cursor.execute.assert_called()
```

Comment:
might be worth testing that it actually did anything interesting too (like make an expected output file)

Reply:
yeah - i sort of treated this as a second class test, since it is just testing a different entrypoint into a method that's tested elsewhere, but lemme try and improve this.

Reply:
I hear ya, and you don't necessarily want to go overboard on an end-to-end unit test. But I like having all the glue between code tested in this way. Which you did! I would just throw a couple more checks in and call it a day.

Reply:
the hard part about this is that the output of all this stuff is 'we executed some queries against athena' - if we had a mock, we could check that data existed.

Reply:
Ah yeah - I live in the privileged ETL world, where I have files on disk.

If it's hard, don't stress about it, just a thought.

Reply:
I did get in here and at least checked to make sure a sensible query exists in the call arg, which did root out an edge case bug, so it was worth doing.

I was thinking about this a little in the background, and I may open up a ticket with BCH IT to try and enable this approach for regression: https://github.com/smart-on-fhir/cumulus-library/issues/110